extensions/amp-analytics/0.1/vendors.js

```diff
@@ -507,6 +507,40 @@ export const ANALYTICS_CONFIG = {
       },
     },
   },
+  'oewadirect': {
+    'vars': {
+      'amp': '1',
```

Comment:
is this value going to be anything other than 1? If not, maybe put it in the url? same below?